files/build_templates/physyncd.service.j2

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=physyncd service
```

Comment:
as we agreed on the db design, we should probably unify the name here, name it as gbsyncd service.

Reply:
Done, renamed.